app/src/main/java/com/ameri/newsapi/util/di/NetworkModule.kt

```diff
@@ -40,7 +37,7 @@ object NetworkModule {
 
     @Provides
     @Singleton
-    fun provideBaseUrl() = BASE_URL
+    fun provideBaseUrl() = BASE_URL_THE_NEWS_API
```

Comment:
Why do you provide the base url with DI?

Reply:
There is no particular reason, I used it based on a tutorial